Pickers/SWActionSheet.h

```diff
@@ -13,7 +13,9 @@
 
 - (void)showFromBarButtonItem:(UIBarButtonItem *)item animated:(BOOL)animated;
 
-- (instancetype)initWithView:(UIView *)view windowLevel:(UIWindowLevel)windowLevel;
+- (instancetype)initWithView:(UIView *)view windowLevel:(UIWindowLevel)windowLevel withSupportedOrientation:(UIInterfaceOrientationMask) mask;
```

Comment:
please, do not change already existing init methind, otherwise we are loosing backward compatability. we have to make new one and make an alias to this.